RundeckAutomation/RundeckAutomation.psd1

```diff
@@ -116,8 +128,7 @@
             Prerelease = ''
 
             # ReleaseNotes of this module
-            ReleaseNotes = @'
-'@
+            ReleaseNotes = 'https://github.com/webmd-health-services/RundeckAutomation/CHANGELOG.md'

```

Comment:
Correct URL is `https://github.com/webmd-health-services/RundeckAutomation/blob/main/CHANGELOG.md` .